dtfs-central-api/src/v1/controllers/tfm/deal/tfm-update-deal.controller.js

```diff
@@ -68,12 +68,23 @@ const updateDeal = async (dealId, dealChanges, existingDeal) => {
   const existingDealActivities = (existingDeal.tfm && existingDeal.tfm.activities);
   const tfmUpdateHasActivities = (tfmUpdate.activities
                                   && Object.keys(tfmUpdate.activities).length > 0);
-
+  /**
+   * ACBS activities update are an array whereas TFM activities update is an object
```

Comment:
Small - update `is` an array whereas TFM `activity`

Reply:
done